src/device.c

```diff
@@ -1,4 +1,7 @@
+/* -*- mode: c; c-file-style: "bsd"; indent-tabs-mode: t; -*- */
```

Comment:
@luka-n, could you remove those? They are editor specific and I don't think they belong in the source file.